depthai_helpers/calibration_utils.py

```diff
@@ -11,6 +11,8 @@
 import cv2.aruco as aruco
 from pathlib import Path
 from collections import deque
+from depthai_helpers.image_scaler import ImageScaler
+
 # Creates a set of 13 polygon coordinates
 traceLevel = 2
```

Comment:
lets add this trave level as a parameter to calibrate.py ?

Reply:
Done. Trace level is now a `StereoCalibration` class member and is passed from `calibrate.py` when the class is constructed.